HW31/task1.py

```diff
@@ -0,0 +1,16 @@
+import requests
+import random
+
+Google = "https://google.com"
+Facebook = "https://facebook.com"
+Twitter = "https://twitter.com"
+Amazon = "https://amazon.com"
+Apple = "https://apple.com"
+
+list_of_sites = [Google, Facebook, Twitter, Amazon, Apple]
```

Comment:
nit: I think it is not necessary to set sites to variables, or if you want it would be better to use dict or object